packages/terra-clinical-result/src/proptypes/statusPropTypes.js

```diff
@@ -5,14 +5,8 @@ import PropTypes from 'prop-types';
  * values mapped from http://hl7.org/fhir/2018Sep/valueset-observation-status.html
  */
 const statusPropType = PropTypes.oneOf([
-  'registered',
-  'preliminary',
   'final',
```

Comment:
I'm not seeing this status type used anywhere. Can this be removed or did I miss where it's being used?

Reply:
those should be removed, got missed.